src/tools/auth0/handlers/logStreams.ts

```diff
@@ -58,9 +58,11 @@ export default class LogStreamsHandler extends DefaultAPIHandler {
       return this.existing;
     }
 
-    const logStreams = await this.client.logStreams.getAll({ paginate: false }).then((logStreams) =>
+    // paginate: false (as not supported by SDK/API)
+    const logStreams = await this.client.logStreams.getAll().then(({ data: logStreams }) =>
       logStreams.map((logStream) => {
-        if (logStream.status === 'suspended') delete logStream.status;
+        // TODO: Do we need to keep this
```

Comment:
Yes, you need to keep this because you cannot manage log streams in a suspended state.